core/src/test/java/apoc/spatial/GeocodeTest.java

```diff
@@ -106,10 +64,6 @@ private void testGeocodeWithThrottling(String supplier, Boolean reverseGeocode,
         assertTrue("Fast " + supplier + " took " + fast + "ms and slow took " + slow + "ms, but expected slow to be at least twice as long", (1.0 * slow / fast) > 1.2);
     }
 
-    private long testGeocode(String provider, long throttle, boolean reverseGeocode) throws Exception {
-        return testGeocode(provider, throttle, reverseGeocode, Collections.emptyMap());
-    }
-
     private long testGeocode(String provider, long throttle, boolean reverseGeocode, Map<String, Object> config) throws Exception {
         setupSupplier(provider, throttle);
 //        testConfig(provider);
```

Comment:
Nothing to do with the PR, but can you remove this comment as well? it looks ugly :P